build_integration_test.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+cli="./faas-cli"
+template="python3"
+
+get_package() {
+    uname=$(uname)
+
+    echo "Getting faas-cli package for $uname..."
+
+    case $uname in
+    "Darwin")
+    cli="./faas-cli-darwin"
+    ;;
+    "Linux")
+        arch=$(uname -m)
+        echo $arch
+        case $arch in
+        "armv6l" | "armv7l")
+        cli="./faas-cli-armhf"
+        template="python3-armhf"
+        ;;
+        esac
+    ;;
+    esac
+
+    echo "Using package $cli"
+    echo "Using template $template"
+}
+
+build_faas_function() {
+
+    function_name=$1
+
+    eval $cli new $function_name --lang $template
+
+cat << EOF > $function_name/handler.py
+def handle(req):
+
+    return "Function output from integration testing: Hello World!"
+EOF
+
+    eval $cli build -f $function_name.yml
+}
+
+wait_for_function_up() {
+    function_name=$1
+    port=$2
+    timeout=$3
+
+    function_up=false
+    for (( i=1; i<=$timeout; i++ ))
```

Comment:
What happened to my suggestion here? I can't see my comment anymore

This would be simpler:

```
for i in {1...10}
```

Reply:
Yep, reply is hidden as outdated. 
`{1...$timeout}` does not work with variables, referring to the docs and after manual testing